source/install/self-managed-changelog.md

```diff
@@ -3,13 +3,115 @@
 
-Lastest Mattermost Releases:
+Latest Mattermost Releases:
+- [Release v6.2 - Feature Release](#release-v6.2-feature-release)
 - [Release v6.1 - Feature Release](#release-v6.1-feature-release)
 - [Release v6.0 - Feature Release](#release-v6.0-feature-release)
 - [Release v5.39 - Quality Release](#release-v5-39-quality-release)
-- [Release v5.38 - Feature Release](#release-v5-38-feature-release)
 - [Release v5.37 - Extended Support Release](#release-v5-37-extended-support-release)
 
+## Release v6.2 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
+
+**Release Day: 2021-12-16**
+
+### Important Upgrade Notes
+ -  Channel results in the channel autocomplete will include private channels. Customers using Bleve or Elasticsearch for autocomplete will have to reindex their data to get the new results. Since this can take a long time, we suggest disabling autocomplete and running indexing in the background. When this is complete, re-enable autocomplete.
+
+**IMPORTANT:** If you upgrade from a release earlier than v6.1, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Playbook Updates
+ - Added Playbooks improvements, including permission schemes, ability to follow runs, and quality of life updates.
+
+#### Boards Updates
+ - Added Boards improvements, including calendar view and @mention autocomplete.
+
+### Improvements
+
+#### User Interface (UI)
+ - Clarified Latex Rendering config setting descriptions and fixed a broken product documentation link.
+ - Updated the "One-click reactions on messages" user setting to "Quick reactions on messages".
+ - Updated **Account Settings** terminology to **Profile**.
+ - Updated instances of **switch** to **navigate**.
+ - Updated in-product text terminology to shift from **comments** to **conversations** and **replies**.
+ - Added a **Click to open thread** setting for all users, to allow users to click anywhere on a message to open the reply thread.
+ - Do Not Disturb option for **Tomorrow** now displays the expiry time.
+ - Recent emojis now get updated based on the default selected skin tone.
+ - Updated **SingleImageView** to hide the image name for attached images until the image is collapsed.
+ - Moved the expand arrow to the left of an image name. 
+ - The image expansion icon now appears on image hover.
+ - Added online status to profile images on user autocomplete.
+ - App Commands now have an option to be opened as modals.
+ - Added support for navigating through Collapsed Reply Threads via arrow keys.
+ - Added support for focusing the input box in Collapsed Reply Threads while typing.
+ - Added support for blurring the input box in Collapsed Reply Threads on pressing Escape.
+ - Adjusted the channel override desktop notification preference for Threads.
+ - User interface is now improved when no text is set for a custom status.
+
+#### Performance
```

Comment:
Do we have any stats on how big these performance improvements are? Should we include in highlights?

Reply:
@hmhealey For any further details here.

Reply:
AFAIK the last two were more performance "fixes" rather than improvements that we can highlight. 

Given there is a lot of performance investigations and ongoing work in that area, I would be hesitant to say we should highlight anything perf related right now

Reply:
Move the last two performance improvements to the Bug Fixes section:

- Removed real-time updates of a couple of features to prevent overloading servers on user updates. The "This channel contains guests" indicator and the number of timezones displayed when notifying members of a group will only be updated on channel change now.
 - Fixed slow channel loading for instances with website link previews enabled.